README.md

```diff
@@ -55,10 +157,28 @@ Contributions, issues, and feature requests are welcome!
 
 Feel free to check the [issues page](../../issues/).
 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please do follow me on social media mentiond above on Authors section.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank <a href="https://www.microverse.org/">Microverse</a> to give me oportunity to work on this project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] As mentioned by the previous reviewer #1 :point_left: kindly add a valid LICENSE. Your `MIT` link is not working so kindly fix it :+1:  Check below :point_down: my suggestions :slightly_smiling_face: 

```suggestion
This project is [MIT](./LICENSE.md) licensed.
```